setup/src/main/java/io/jenkins/jenkinsfile/runner/JenkinsEmbedder.java

```diff
@@ -322,7 +322,7 @@ protected Hudson newHudson() throws Exception {
             r.decorateHome(this, home);
         }
         try {
-            return new Hudson(home, webServer, getPluginManager());
+            return new Jenkins(home, webServer, getPluginManager());
```

Comment:
```suggestion
            return new Hudson(home, webServer, getPluginManager());
```

This causes compilation failure

Reply:
I am starting a clean slate now and checking to all the errors that I made and resolving it.